h2o-core/src/main/java/hex/ModelMetricsBinomial.java

```diff
@@ -272,8 +272,12 @@ private GainsLift calculateGainsLift(Model m, Frame preds, Vec resp, Vec weights
       GainsLift gl = null;
       try {
         gl = new GainsLift(preds.lastVec(), resp, weights);
+        if (m._parms._gainslift_bins !=0 ) {
+          if (m._parms._gainslift_bins < 1) throw new IllegalArgumentException("Number of G/L bins must be greater than zero.");
+          gl._groups = m._parms._gainslift_bins;
```

Comment:
interesting that this was there but was actually never used :)

Reply:
Yes, truly amazing.